pkg/storage/ingest/fetcher.go

```diff
@@ -694,8 +698,11 @@ func (w *inflightFetchWants) removeNextResult() {
 	w.wants.Remove(head)
 }
 
-func (r *concurrentFetchers) start(ctx context.Context, startOffset int64, concurrency, recordsPerFetch int) {
-	level.Info(r.logger).Log("msg", "starting concurrent fetchers", "start_offset", startOffset, "concurrency", concurrency, "recordsPerFetch", recordsPerFetch)
+func (r *concurrentFetchers) start(ctx context.Context, startOffset int64, concurrency int) {
+	const defaultBytesPerRecord = 10_000 // start with an estimation, we will update it as we consume
```

Comment:
[nit] Don't hate me. I think "initial" is more accurate than "default". This is not a fallback, it's just the initial value, before we can estimate.

```suggestion
	const initialBytesPerRecord = 10_000 // start with an estimation, we will update it as we consume
```